qvm-tools/qvm-ls

```diff
@@ -24,7 +24,7 @@
 from qubes.qubes import QubesVmCollection
 from qubes.qubes import QubesHost
 from qubes.qubes import QubesException
-from optparse import OptionParser
+from argparse import ArgumentParser
```

Comment:
For the next time: please make such changes as a separate commit - one for changing library but without any functional change, then actual change. It will make review much easier.